packages/core/src/test/sequence-validation.test.tsx

```diff
@@ -21,10 +21,12 @@ test('It should allow null as children', () => {
 	).not.toThrow();
 });
 
-test('It should not allow no children at all', () => {
-	expectToThrow(
-		// @ts-expect-error
-		() => render(<Sequence from={0} />),
-		/You passed to durationInFrames an argument of type undefined, but it must be a number./
-	);
+test('It should allow undefined as children', () => {
+	expect(() =>
+		render(
+			<Sequence durationInFrames={100} from={0}>
+				{undefined}
+			</Sequence>
+		)
+	).not.toThrow();
```

Comment:
Can you add the previous test back? It tested for when you were missing the `durationInFrames` property. I know the test name was bad, maybe we can change it to "Should enforce durationInFrames props"

Reply:
Previously this third test was an exact duplicate of the first test. Now, it's testing something different. 

The name of the first test should be explanation enough, I think: 'It should throw if Sequence has missing duration'

Reply:
You are right! Thanks for correcting that.